app/src/docs/_examples/richTextEditor/Serialization.example.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+import { Panel, FlexRow, Button, RichTextView } from '@epam/promo';
+import {
+    SlateEditor,
+    imagePlugin,
+    videoPlugin,
+    baseMarksPlugin,
+    linkPlugin,
+    iframePlugin,
+    quotePlugin,
+    superscriptPlugin,
+    headerPlugin,
+    listPlugin,
+    EditorValue,
+    codeBlockPlugin,
+    createSerializer,
+    isEditorValueEmpty,
+    migrateSchema,
+    createDeserializer,
+    paragraphPlugin,
+} from '@epam/uui-editor';
+import { demoData } from '@epam/uui-docs';
+import css from './SlateEditorBasicExample.module.scss';
+
+const plugins = [
+    paragraphPlugin(),
+    baseMarksPlugin(),
+    headerPlugin(),
+    superscriptPlugin(),
+    listPlugin(),
+    quotePlugin(),
+    linkPlugin(),
+    imagePlugin(),
+    videoPlugin(),
+    iframePlugin(),
+    codeBlockPlugin(),
+];
+
+const serializeHTML = createSerializer();
+const deserializeHTML = createDeserializer();
+
+export default function SlateEditorBasicExample() {
+    const [value, setValue] = useState<EditorValue>(migrateSchema(demoData.slateSerializationInitialData));
+
+    const [serializedHtml, setSerializedHtml] = useState(serializeHTML(value));
+    const [type, setType] = React.useState<'html' | 'edit'>('edit');
+
+    const onChangeEditorValue = (newValue: EditorValue) => {
+        setValue(newValue);
+    };
+
+    const onView = () => {
+        setSerializedHtml(serializeHTML(value));
+        setType('html');
+    };
+
+    const onEdit = () => {
+        setValue(deserializeHTML(serializedHtml));
+        setType('edit');
+    };
+
+    return (
+        <Panel cx={ css.root }>
+            <FlexRow spacing="18" vPadding="12">
+                {type === 'edit' && (
```

Comment:
Let's better use Switch component, like we have at another RTE example
<img width="145" alt="image" src="https://github.com/epam/UUI/assets/14283670/8373dd81-5f1d-45a3-ab6a-e5273a307b2e">

Reply:
Replaced